<?xml version="1.0" encoding="utf-8"?>
<TestRun id="e866a533-15d2-44fd-87b5-a7e0dfd489a0" name="anna@cloudtest-win10 2023-01-22 23:33:55" runUser="cloudtest-win10\anna" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2023-01-22T23:33:55.4600667+00:00" queuing="2023-01-22T23:33:55.4600673+00:00" start="2023-01-22T23:33:48.8337843+00:00" finish="2023-01-22T23:33:55.5049234+00:00" />
  <TestSettings name="default" id="f3b24d3d-905d-4b92-9ce3-f8835b341231">
    <Deployment runDeploymentRoot="anna_cloudtest-win10_2023-01-22_23_33_55" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="292eb8ba-b6d7-4a3a-8428-29f2a9520ddb" testId="f7e60991-4842-8663-713d-bb1731c8ccbe" testName="Tester.AzureUtils.AzureTableDataManagerTests.AzureTableDataManager_CreateTableEntryAsync" computerName="cloudtest-win10" duration="00:00:01.7317412" startTime="2023-01-22T23:33:55.1557092+00:00" endTime="2023-01-22T23:33:55.1557553+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="292eb8ba-b6d7-4a3a-8428-29f2a9520ddb" />
  </Results>
  <TestDefinitions>
    <UnitTest name="Tester.AzureUtils.AzureTableDataManagerTests.AzureTableDataManager_CreateTableEntryAsync" storage="c:\users\anna\orleans\test\extensions\testerazureutils\bin\debug\net7.0\tester.azureutils.dll" id="f7e60991-4842-8663-713d-bb1731c8ccbe">
      <Execution id="292eb8ba-b6d7-4a3a-8428-29f2a9520ddb" />
      <TestMethod codeBase="C:\Users\anna\orleans\test\Extensions\TesterAzureUtils\bin\Debug\net7.0\Tester.AzureUtils.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="Tester.AzureUtils.AzureTableDataManagerTests" name="AzureTableDataManager_CreateTableEntryAsync" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="f7e60991-4842-8663-713d-bb1731c8ccbe" executionId="292eb8ba-b6d7-4a3a-8428-29f2a9520ddb" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Completed">
    <Counters total="1" executed="1" passed="1" failed="0" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.5+1caef2f33e (64-bit .NET 7.0.2)&#xD;
[xUnit.net 00:00:01.27]   Discovering: Tester.AzureUtils (method display = ClassAndMethod, method display options = None)&#xD;
[xUnit.net 00:00:01.48]   Discovered:  Tester.AzureUtils (found 246 test cases)&#xD;
[xUnit.net 00:00:01.50]   Starting:    Tester.AzureUtils (parallel test collections = off, max threads = 4)&#xD;
[xUnit.net 00:00:03.33]   Finished:    Tester.AzureUtils&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="cloudtest-win10" outcome="Warning" timestamp="2023-01-22T23:33:55.3118705+00:00">
        <Text>Data collector 'Blame' message: All tests finished running, Sequence file will not be generated.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>